cardinal/system/system_game_setup.go

```diff
@@ -100,6 +104,15 @@ func (s *GameSetup) setupPlain() {
 		"the air tastes of burnt grease and bensons.",
 	)
 
+	/*  LOGS FOR TESTING VERIFICATION!
+	// Logs to verify the ID's in the setup
+	log.Printf("Object IDs for room %d: %v", roomID, objs)
```

Comment:
you can change the logs to something like slog which allows you to have levels of loggind like INFO WARN ERROR etc then you don't need to comment things out

Reply:
I just placed this ones for testing the results on the console. Once it is not require I will remove them from the code.

Reply:
Sorry, forgot to remove those. Will do now

Reply:
comments

Reply:
done